docs/source/yelpsoa_configs.rst

```diff
@@ -111,6 +111,8 @@ instance MAY have:
     Parsing the Marathon config file will fail if both args and cmd are
     specified [#note]_.
 
+  .. _env:
```

Comment:
Ooo. Is there any way to get automatic anchors like that? Even if not, go for it. (I love permalinks)